packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTask.js

```diff
@@ -11,17 +13,44 @@ function SurveyTask (props) {
     task
   } = props
 
+  const [selectedChoice, setSelectedChoice] = useState(false)
+
   function handleChoice (selectedChoiceId) {
     console.log('Selected choiceId =', selectedChoiceId)
+
+    setSelectedChoice(selectedChoiceId)
   }
 
+  const choiceTargetRef = useRef()
+
   return (
-    <Chooser
-      autoFocus={autoFocus}
-      disabled={disabled}
-      task={task}
-      onChoose={handleChoice}
-    />
+    <Box
+      ref={choiceTargetRef}
+    >
+      <Chooser
+        autoFocus={autoFocus}
+        disabled={disabled}
+        onChoose={handleChoice}
+        task={task}
+      />
+      {choiceTargetRef.current && selectedChoice && (
+        <Drop
+          align={{
+            top: 'top'
```

Comment:
`<Drop align={{ top: 'top' }} />` is not producing the expected result. I thought this would position top of the drop component aligned with the top of the `target={choiceTargetRef.current}` component, but instead it appears that the bottom of both components are aligned.

Reply:
I think this is functioning as intending as far as I can see. Here's what I'm seeing:

<img width="419" alt="Screen Shot 2021-02-11 at 2 03 04 PM" src="https://user-images.githubusercontent.com/5016731/107692065-e4c2ad80-6c71-11eb-9acf-ce6c3ee8cb16.png">

It appears to be top aligned to me minus the padding of the Box?

Now that I've seen it, though, I'm not sure if Drop is the best UI for this versus how it rendered in PFE just inline. It feels a bit awkward to me to have it be modal-like and appear to be on top. @beckyrother thoughts? (Totally ok if you two have already talked about this and this is the intention. Just let me know).

Reply:
@srallen not sure what you mean?

Reply:
@beckyrother I'll make a gif to illustrate it.

Reply:
I don't have a question about this anymore, not sure why I was seeing what I did when originally commented, but it's as expected now.